apps/labs/posts/improving-accessibility-in-jupyterlab.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Improving Accessibility in JupyterLab'
+author: kulsoom-zahra
+published: October 31, 2022
+description: 'blog post sharing my experience working as a Quansight labs intern'
+category: [Access-centered, OSS Experience]
```

Comment:
I'm not sure what the category `OSS Experience` is for. Does this blog post fall within that category?

Reply:
As far as I know, and the reason I put that tag is it means Open-Source Software.